apps/compilersforhumans/src/pages/index.tsx

```diff
@@ -1,18 +1,25 @@
 import * as React from 'react'
-// import ConvertkitSubscribeForm from '@skillrecordings/convertkit/dist/forms'
 import ConvertkitSubscribeForm from 'components/forms/convertkit'
 import Layout from 'layouts'
 import LandingCopy from 'components/landing-copy.mdx'
-import Animation from 'components/animation'
+import Blob from 'components/blob'
 import {useRouter} from 'next/router'
+import {WEBGL} from 'utils/webgl'
 
 export default function Home() {
   const router = useRouter()
+  const [hasWebGLSupport, setWebGLSupport] = React.useState<boolean>(false)
+  React.useEffect(() => {
+    setWebGLSupport(WEBGL.isWebGLAvailable())
```

Comment:
I was able to reproduce the `Failed to create WebGL context` message mentioned in #79 by disabling WebGL support in Firefox. By adding this check we can skip rendering canvas and thus avoid getting whole site to error out.

I think there might be more going on for [this user](https://twitter.com/tburgess57/status/1436507920896757765/photo/1) though as some of their warning messages mention GL Drivers and system restrictions. Would be interesting to see if this fixes it for them.